tutorials/nemo-retriever-synthetic-data-generation/retriever_hardnegative_miner.py

```diff
@@ -0,0 +1,306 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+import itertools
+import pdb
```

Comment:
Remove this 
```suggestion
import pdb
```

Broadly there are a bunch of unsued imports , so probably we can do below to remove them. 

```
pip install flake8 flake8-unused-imports

```

```
flake8 your_code.py

```

Reply:
done